tests/AnalyticsGdpr/AnalyticsGdprCliTest.php

```diff
@@ -59,7 +59,6 @@
 
 	$descKey = 'shortdesc';
 
-	$this->assertIsArray($documentation);
-	$this->assertArrayHasKey($descKey, $documentation);
-	$this->assertSame('Generates project Analytics and GDPR Settings classes using ACF.', $documentation[$descKey]);
+	expect($documentation)->toBeArray()->toHaveKey($descKey);
+	expect($documentation[$descKey])->toBeString()->toBe('Generates project Analytics and GDPR Settings classes using ACF.');
```

Comment:
```suggestion
	expect($documentation)
		->toBeArray()
		->toHaveKey($descKey);
	
	expect($documentation[$descKey])
		->toBeString()
		->toBe('Generates project Analytics and GDPR Settings classes using ACF.');
```

I like to put multiple chained expectations one after another, makes it a bit more readable 🙂